src/core/mcis/control.go

```diff
@@ -784,7 +793,7 @@ func CheckAllowedTransition(nsId string, mcisId string, action string) error {
 
 	UpdateMcisInfo(nsId, mcisTmp)
 
-	if strings.Contains(mcisStatusTmp.Status, StatusTerminating) || strings.Contains(mcisStatusTmp.Status, StatusResuming) || strings.Contains(mcisStatusTmp.Status, StatusSuspending) || strings.Contains(mcisStatusTmp.Status, StatusCreating) || strings.Contains(mcisStatusTmp.Status, StatusRebooting) {
+	if strings.Contains(mcisStatusTmp.Status, StatusCreating) || strings.Contains(mcisStatusTmp.Status, StatusTerminating) || strings.Contains(mcisStatusTmp.Status, StatusResuming) || strings.Contains(mcisStatusTmp.Status, StatusSuspending) || strings.Contains(mcisStatusTmp.Status, StatusRebooting) {
```

Comment:
여기는 뭔가 더 좋은 방법이 있을 수도 있을 것 같은데.. 바로 생각은 안 나네요.. 😅

아래는 가독성 및 코드 유지보수 편의성 관련 입니다.. 😊
(실제로 빌드해 보지는 않아서.. Syntax error 날 수도 있겠네요..
아니면 gofmt 가 자동으로 원복시키거나.. 😅)

```suggestion
	if strings.Contains(mcisStatusTmp.Status, StatusCreating) || 
	strings.Contains(mcisStatusTmp.Status, StatusTerminating) || 
	strings.Contains(mcisStatusTmp.Status, StatusResuming) || 
	strings.Contains(mcisStatusTmp.Status, StatusSuspending) || 
	strings.Contains(mcisStatusTmp.Status, StatusRebooting) {
```